integration/test_schema.py

```diff
@@ -92,3 +93,30 @@ def test_schema_keys(client: weaviate.Client):
     }
     client.schema.create_class(single_class)
     assert client.schema.exists("Author")
+
+
+def test_class_tenants(client: weaviate.Client):
+    class_name = "MultiTenancy"
+    single_class = {
+        "class": class_name,
+        "properties": [
+            {
+                "dataType": ["text"],
+                "name": "author",
+            }
+        ],
+        "multiTenancyConfig": {"enabled": True, "tenantKey": "author"},
+    }
+    client.schema.create_class(single_class)
+    assert client.schema.exists(class_name)
+
+    tenants = [
+        Tenant(name="Tenant1"),
+        Tenant(name="Tenant2"),
+        Tenant(name="Tenant3"),
+        Tenant(name="Tenant4"),
+    ]
+    client.schema.create_class_tenants(class_name, tenants)
+
+    exists = client.schema.exists(class_name)
+    assert exists
```

Comment:
What would be failure modes that we could test?
- Tenant exists (eg adding two times)
- wrong type
- what happens when I call add tenant on a class without tenant config?
- what happens if I call a tenant class without tenant argument?